go-controller/pkg/cni/cni.go

```diff
@@ -195,20 +196,26 @@ func (pr *PodRequest) cmdDel(clientset *ClientSet) (*Response, error) {
 					pr.PodNamespace, pr.PodName, pr.nadName, err)
 			} else {
 				ovsIfName := ovsIfNames[0]
-				out, err := ovsGet("interface", ovsIfName, "external_ids", "vf-netdev-name")
+				out, err := ovsGet("interface", ovsIfName, "external_ids", "netdev-name")
 				if err != nil {
-					klog.Warningf("Couldn't find the original VF Netdev name from OVS interface %s for pod %s/%s: %v",
-						ovsIfName, pr.PodNamespace, pr.PodName, err)
+					// Check if interface doesn't have legacy vf-netdev-name
```

Comment:
Does this mean that you can have both vf-netdev-name and netdev-name with backwards compatibility ? Also netdev-name can't be a VETH right?